pinot-core/src/main/java/org/apache/pinot/core/operator/dociditerators/MVScanDocIdIterator.java

```diff
@@ -47,15 +47,17 @@ public final class MVScanDocIdIterator implements ScanBasedDocIdIterator {
 
   private int _nextDocId = 0;
   private long _numEntriesScanned = 0L;
+  private final int _batchSize;
 
-  public MVScanDocIdIterator(PredicateEvaluator predicateEvaluator, DataSource dataSource, int numDocs) {
+  public MVScanDocIdIterator(PredicateEvaluator predicateEvaluator, DataSource dataSource, int numDocs, int batchSize) {
```

Comment:
Custom batch size might not apply here since this batch is only used when iterating over the bitmap, where 256 might be the optimal one. It won't reduce the time of index access

Reply:
You are right, although I guess we will need to modify `MVScanDocIdIterator` in future in order to optimize the access in this case.